StudCity.Frontend/src/UI/IconButton.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 const IconButton = ({ children, className = "", ...custom }) => {
 	return (
 		<button
-			className={`duration-500 rounded-full w-12 h-12 p-1 hover:bg-primaryGold text-white ${className}`}
+			className={`duration-500 rounded-full w-12 h-12 p-1 bg-primaryGold text-white ${className}`}
```

Comment:
You removed the word `hover:` it can break the logic of other buttons, I made them transparent on purpose, and you can change the color from the parent component